petabvis/window_functionality.py

```diff
@@ -207,7 +248,7 @@ def exchange_dataframe_on_click(index: QtCore.QModelIndex,
         if window.observable_df.equals(df):
             df_changed = False
         window.observable_df = df
-    if parent_name == "simulation_files":
+    if parent_name == "Simulation Table":
```

Comment:
Can this be changed to a constant? I think one already exists?

Reply:
I do not know any constant that already exists. I think petab.C has no constant for the simulation file

Reply:
Ah, I meant something like below (which could also be moved to the constants file, and changed to a class, with an additional value for simulation tables).

https://github.com/PEtab-dev/petabvis/blob/727d56f852c2ac78046b2ac51514f1b2f9e6e2ca/petabvis/window_functionality.py#L103-L107